### ISSUE TITLE
`includeClasses` keys are case-sensitive

### ISSUE THREAD

**Issue by albe** [neos/flow-development-collection#1784]
Not sure why this is the case, as it seems to not make much sense, but causes hard to debug issues when a (non-flow/neos) package is not object managed, even though it's explicitly configured in Settings.

Questions to solve:
- How is the packageKey in `PackageInformationCache` built? In most cases it seems to be lowercase but not every time.
- why isn't the includeClasses key check done case-insensitive inside `CompileTimeObjectManager::registerClassFiles`? Is it already expected to all be in a normalized (lower) casing?

**Comment by bwaidelich** [comment]
Which `includeClasses` are we talking about?

**Comment by albe** [comment]
There are multiple? 😱 This is about `Neos.Flow.object.includeClasses`

**Comment by bwaidelich** [comment]
Ah of course the setting! I was just skimming through the description and didn't understand what this issue is about :)

So the problem is that
```yaml
Neos:
  Flow:
    object:
      includeClasses:
        'Some.PackageKey' : ['.*']
```
won't work if the corresponding package key is not `Some.PackageKey` but `Some.Packagekey` for example?

**Comment by albe** [comment]
Yeah, basically that. It came up in slack the day I created this issue (see https://neos-project.slack.com/archives/C04V4C6B0/p1568972029033900) and it was rather the other way around.

The package was named `Some.Name` and the `includeClasses` contained `some.name` (probably because the composer package name is `some/name` lowercase too).
